runtime/interpreter/simplecompositevalue.go

```diff
@@ -56,10 +56,8 @@ func NewSimpleCompositeValue(
 	stringer func(SeenReferences) string,
 ) *SimpleCompositeValue {
 
-	baseUsage, dataUsage, metaDataUsage := common.NewCompositeMemoryUsages(uint64(len(fields)), 0)
-	common.UseMemory(inter, baseUsage)
-	common.UseMemory(inter, dataUsage)
-	common.UseMemory(inter, metaDataUsage)
+	common.UseMemory(inter, common.NewConstantMemoryUsage(common.MemoryKindCompositeBase))
```

Comment:
Is the base same for both composites and simple-composites? The fields of the two go-structs are different, so the weights also need to be different.

Also, the base memory-usage (i.e: `common.NewConstantMemoryUsage(common.MemoryKindCompositeBase)`) can be pre computed / 
 computed only once